psd-web/app/models/user.rb

```diff
@@ -151,6 +151,10 @@ def is_team_admin?
     has_role? :team_admin
   end
 
+  def has_setup_account?
```

Comment:
```suggestion
  def has_completed_registration?
```
Just to align the language through the service and avoid confusions like the `account_activated?` one.

I would also rename the `account_activated` flag as is clearly ambiguous and prone to confusions (as happened to us).

Reply:
I think this is better, although "registration" is also arguably ambiguous (does registration include accepting the declaration or not?).

Maybe to be super explicit we could call it: `has_login_credentials?` or even `has_created_password_and_two_factor_authentication`?

Reply:
I like the `has_login_credentials?` one.

Reply:
I like `has_completed_registration` since we have decided to separate the declaration from registration.

Reply:
ok let's go with `has_completed_registration?`